.pre-commit-config.yaml

```diff
@@ -26,8 +26,11 @@ repos:
   rev: v2.34.0
   hooks:
   - id: pyupgrade
-- repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v0.961
+- repo: local
```

Comment:
please use the additional dependencies for the pre-commit mirros mypy hook

Reply:
Is it possible to add the local repo to the dependencies? local `mypy` and mirrored `mypy` provides different results due to missing `webdriver_kaifuku` in the virtual environment.

Reply:
https://mypy.readthedocs.io/en/stable/config_file.html#confval-mypy_path sorts exactly this